FrontEnd/src/components/MakeAPost/Page3.tsx

```diff
@@ -6,16 +6,30 @@ import {
   Tooltip,
   WizardFormStep,
 } from '@basics';
-import { Month, months, NON_EMPTY_ERR_MSG } from '@constants';
-import { runNTimes } from '@utils';
+import { NON_EMPTY_ERR_MSG } from '@constants';
 import cn from 'classnames';
 import moment from 'moment';
-import React, { FunctionComponent, useEffect, useRef, useState } from 'react';
+import React, { FunctionComponent, useEffect, useRef } from 'react';
 import * as z from 'zod';
 import styles from './Page.module.scss';
 
 export type Page3Store = z.infer<typeof page3Schema>;
 
+const month = [
```

Comment:
months are already created inside of constants somewhere (potentially as an enum)